### PR TITLE
Quick code fixes in webapp

### PR DESCRIPTION
**Pull request by latin-panda**
# Description
Sending these quick fixes in a separate PR for fast merging into the main branch, because I'm working on a feature release that might take long to merge into main. 

This PR:
- Fixes data types in Enketo Service interface.
- Remove unused parameters in Enketo Service function.
- Fixes alias in imports.

Ticket: https://github.com/medic/cht-core/issues/7557

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
@dianabarsan can you please have a look at this small fixes? I'm guessing we don't need to AT these, right?

File: webapp/src/ts/modules/configuration-user/configuration-user.routes.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import {Routes} from '@angular/router';
-import {ConfigurationUserComponent} from './configuration-user.component';
+import { Routes } from '@angular/router';
+
+import { ConfigurationUserComponent } from './configuration-user.component';
```
Comment (dianabarsan) [inline webapp/src/ts/modules/configuration-user/configuration-user.routes.ts:3]:
```suggestion
import { ConfigurationUserComponent } from '@mm-modules/configuration-user/configuration-user.component';
```

File: webapp/src/ts/modules/about/about.routes.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
-import {Routes} from '@angular/router';
-import {AboutComponent} from './about.component';
+import { Routes } from '@angular/router';
+
+import { AboutComponent } from './about.component';
```
Comment (dianabarsan) [inline webapp/src/ts/modules/about/about.routes.ts:3]:
```suggestion
import { AboutComponent } from '@mm-modules/about/about.component';
```

File: webapp/src/ts/modules/error/error.routes.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import { Routes } from '@angular/router';
+
 import { ErrorComponent } from './error.component';
```
Comment (dianabarsan) [inline webapp/src/ts/modules/error/error.routes.ts:3]:
```suggestion
import { ErrorComponent } from '@mm-modules/error/error.component';
```

File: webapp/src/ts/modules/home/home.routes.ts [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import { Routes } from '@angular/router';
+
 import { HomeComponent } from './home.component';
```
Comment (dianabarsan) [inline webapp/src/ts/modules/home/home.routes.ts:3]:
```suggestion
import { HomeComponent } from '@mm-modules/home/home.component';
```

**Review by dianabarsan — CHANGES_REQUESTED** [review]
Just a few stragglers. 
If it were just for the changed paths, I would agree that this doesn't need AT (since app building should fail if any of the paths is incorrect). 
However, making the change in the EnketoService interface does make AT required, since that is a functional change, and I don't think we cover forms with media elements in our e2e tests.

File: webapp/src/ts/modules/configuration-user/configuration-user.routes.ts (hunk 1 above)
Comment (latin-panda) [reply webapp/src/ts/modules/configuration-user/configuration-user.routes.ts:3]: Done

File: webapp/src/ts/modules/about/about.routes.ts (hunk 2 above)
Comment (latin-panda) [reply webapp/src/ts/modules/about/about.routes.ts:3]: Done

File: webapp/src/ts/modules/error/error.routes.ts (hunk 3 above)
Comment (latin-panda) [reply webapp/src/ts/modules/error/error.routes.ts:3]: Done

File: webapp/src/ts/modules/home/home.routes.ts (hunk 4 above)
Comment (latin-panda) [reply webapp/src/ts/modules/home/home.routes.ts:3]: Done

**Comment by latin-panda** [comment]
Thanks @dianabarsan, I've applied the suggestions and created a ticket for better AT tracking. Please have a look again when you have time :)

**Review by dianabarsan — APPROVED** [review]
🌟

**Review by dianabarsan — APPROVED** [review]
🌟

**Pull request merged**